client/src/pages/Doctors.tsx

```diff
@@ -1,7 +1,20 @@
-const Doctors = () => (
-  <h1>
-    we are doctors
-  </h1>
-);
+import Stack from '@mui/material/Stack';
+import { useEffect, useState } from 'react';
+import Doctor from '../components/Doctor';
+import IUser from '../Interfaces/post/IUser';
+import ApiServices from '../services/ApiService';
+
+const Doctors = () => {
+  const [doctors, setDoctors] = useState([]);
+  useEffect(() => {
+    ApiServices.get('doctors').then((data) => setDoctors(data.data));
```

Comment:
Are you guys using promises or async await? So much patterns leads to very bad hard to maintain code